src/main/java/com/eppo/sdk/EppoClient.java

```diff
@@ -63,6 +81,12 @@ public static class Builder {
     private long pollingIntervalMs = DEFAULT_POLLING_INTERVAL_MS;
     private String host = DEFAULT_HOST;
 
+    // Assignment and bandit caching on by default. To disable, call
+    // `builder.assignmentCache(null).banditAssignmentCache(null);`
+    private IAssignmentCache assignmentCache = new LRUInMemoryAssignmentCache(100);
```

Comment:
How much of this configuration should we surface to devs?
The cache size or ttl? or the `IAssignmentCache`. Or should we surface as a set of caching options on the builder, similar to JS (ex: `withLruAssignmentCaching(100)`, `withExpiringCache(10, MINUTES)` etc)

Reply:
I like the option of providing convenience methods like `withLruAssignmentCaching` and `withExpiringCache`, however we should also allow users to bring their own implementation of `IAssignmentCache`, so we also need a `withAssignmentCache` that takes the interface. EDIT: Looks like you already added it below, so we're good on that one